code/modules/asset_cache/assets/light_templates.dm

```diff
@@ -0,0 +1,7 @@
+/datum/asset/spritesheet/lights
+	name = "lights"
+
+/datum/asset/spritesheet/lights/create_spritesheets()
+	for(var/id in GLOB.light_types)
+		var/datum/light_template/template = GLOB.light_types[id]
+		Insert("light_fantastic_[template.id]", template.icon, template.icon_state)
```

Comment:
Need to figure out the white backgrounds, if anyone knows how to bypass those please tell me

Reply:
I hacked it, but it does work so eh